### ISSUE TITLE
Very Slow Read-depth annotation Step

### ISSUE THREAD

**Issue by jjfarrell** [dellytools/delly#92]
I am running Delly (recently github cloned version)  on about 800 crams.  For about 200 of them, there is little progress  during the last two days for the Read Depth Annotation Step.  Should I wait this out or does this look like a bug? Any suggestions?

Here is the output......

```
[2018-Jan-09 18:57:06] Paired-end clustering
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 05:38:54] Split-read alignment

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 08:58:46] Junction read annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 09:28:17] Breakpoint spanning coverage annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 12:46:07] Read-depth annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
**********
```

**Comment by tobiasrausch** [comment]
The read-depth annotation so far used the BAM index to jump to every SV and that indeed can get very slow. I now switched to a method that just streams the BAMs once. That should be a lot faster for large number of SVs.

**Comment by jjfarrell** [comment]
Wow!  There was a huge speed-up with the streaming.  Thanks. 

Below is the progress output from rerunning Delly on one of the slow running CRAMs. I will check it out on the complete set of 800 next.  Are there plans for another release soon?

[2018-Jan-09 18:57:13] Paired-end clustering

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 01:41:12] Split-read alignment

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 05:46:16] Junction read annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 06:39:37] Breakpoint spanning coverage annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************
[2018-Jan-10 09:07:31] Read-depth annotation

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|

**Comment by tobiasrausch** [comment]
I just released Delly v0.7.8 which should fix this issue. If you still experience performance issues please let me know.
